src/client/components/RawLink.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useVariation, Variation } from 'utils/hooks/useVariation'
+
+type RawLinkProps = {
+  className?: string
+  external?: boolean
+  href: string
+}
+
+export const RawLink: React.FC<RawLinkProps> = ({
+  className,
+  children,
+  external,
+  href,
+}) => {
+  const variation = useVariation()
+  const externalCondition =
+    external && ![Variation.IOS, Variation.ANDROID].includes(variation!)
```

Comment:
Why do we need the non-null assertion here for `variation`? Always makes me worried 😅

Reply:
Hehe, yeah not the best feeling :D